pkg/config/cluster.go

```diff
@@ -69,7 +79,7 @@ func (c *Cluster) TLS() TLS {
 		return TLS{
 			Insecure:    true,
 			RootCAsPath: tlsVal,
-		}
+		}, fmt.Errorf("can't read %s: %s", tlsVal, err)
```

Comment:
This error starts with "can't" but the one below starts with "cannot", let's have both of them starting with "cannot" as it's the most common form in our error messages.

Reply:
Fixed it with specific error. User will get clear message with instruction what to do